ccm_web/server/src/main.ts

```diff
@@ -72,11 +73,16 @@ async function bootstrap (): Promise<void> {
 
   const configService = app.get<ConfigService<Config, true>>(ConfigService)
   const serverConfig = configService.get('server', { infer: true })
+  const privacyURL = configService.get('privacyURL', { infer: true })
 
   app.set('trust proxy', 1)
 
   doAppCoreSetup(app, serverConfig)
 
+  app.use('/privacy', (req: Request, res: Response) => {
+    res.redirect(privacyURL);
+  });
```

Comment:
Why is this URL added to ccm? isn't it should part of banner code to open in a new window?

Reply:
The privacy path is a part of the documentation for the Umich Onetrust consent banner, it simply directs to <domain>/privacy/. I had to add this as environment variables to MyLA and ROHQ as well. https://vpcomm.umich.edu/resources/cookie-disclosure/